test_local_changes.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+import os
+import shutil
+
+import Bundle_ASH_script
+
+# If needed, replace this with the path to your mafia's base config folder
+MAFIA_DIRECTORY = f"/Users/{os.getlogin()}/Library/Application Support/Kolmafia"
+
+# Change this if you prefer a different name in-game
+BUNDLED_SCRIPT_NAME = "relay_TourGuide-Dev.ash"
+
+if not os.path.exists(MAFIA_DIRECTORY):
+    print(f"""Path {MAFIA_DIRECTORY} doesn't exist.
+Try editing MAFIA_DIRECTORY in {os.path.basename(__file__)} to your mafia's base config folder.
+""")
+    exit(1)
+
+if os.path.exists(BUNDLED_SCRIPT_NAME):
+    os.remove(BUNDLED_SCRIPT_NAME)
+
+Bundle_ASH_script.bundle_and_write("relay/relay_TourGuide.ash", BUNDLED_SCRIPT_NAME, "Source")
```

Comment:
one quick question on this. any reason we aren't doing a TS style bundle like what gets run when it gets packaged to release? i am actually (largely) a python dev so this workflow makes sense to me but i think we already have a bundler, exposing the one we already have might be better than this even.

Reply:
No real reason, I'd actually prefer something more TS-style if there's a good example of that! I just modeled this after what the GitHub action is doing. https://github.com/loathers/TourGuide/blob/main/.github/workflows/main.yml#L317